core/creators/themedui.js

```diff
@@ -384,6 +388,20 @@ CKEDITOR.replaceClass = 'ckeditor';
 		return editor;
 	}
 
+	function delayCreation( element, config, data, mode ) {
+		if ( config.delayDetached && element.isDetached() ) {
+			var intervalId = setInterval( function() {
+				if ( !element.isDetached() ) {
+					createInstance( element, config, data, mode );
+
+					clearInterval( intervalId );
```

Comment:
I guess it will be better to move `clearInterval()` call to top so there is no risk of calling `createInstance` more than one time in some strange race condition, small interval cases.

Reply:
Moved :)